.github/ISSUE_TEMPLATE/release_tracking.md

```diff
@@ -0,0 +1,80 @@
+---
+name: 🚋 Release cycle tracking
+about: Create a new release cycle tracking issue for a minor release
+title: Tasks for v<release-tag> release cycle
+labels: ''
+assignees: ''
+
+---
+
+Please see the corresponding section in [release-tasks.md](https://github.com/sbueringer/cluster-api/blob/pr-release-tasks/docs/release/release-tasks.md) for documentation of individual tasks.  
```

Comment:
Sorry if there was a reason discussed and I missed it, but shouldn't these point to the main repo and not your fork? Could be updated later once the docs are in place, but would be good to make sure the local references are correct.

Reply:
No worries. Yup they should point to the main repo and I'm tracking it as an immediate follow-up to this PR.

I just added the links like this so they can be already verified by the linkchecker and reviewed. It's then only a simple search/replace to change the repo. 
(relative links don't work in issue templates / issues)

Reply:
> but would be good to make sure the local references are correct.

Not sure I understand what you mean.

Reply:
I meant local as far as doing something like:

```markdown
[release-tasks.md](../blob/pr-release-tasks/docs/release/release-tasks.md)
```

But in an issue template it probably is better to have the fully qualified URL instead. So please ignore. :)

Reply:
Hm nice. That actually works, didn't really think about it :) 

I think the full qualified URL has the advantage that the link checker has a chance to validate it, but no strong opinion.

But yup I'll leave it as is